.github/workflows/test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        ruby: [2.5, 2.6, 2.7, 3.0, 'head']
+        ruby: [2.5, 2.6, 2.7, '3.0', 'head']
```

Comment:
Could we add `3.1` here as well? This way it is clear what versions are currently supported  
Also, let's use `3.0` instead of `'3.0'`

Reply:
@timfjord Neither of those changes work.

Regarding the first, this is a matrix.  Ruby 3.1 is not supported against the Rails versions in your matrix.  For example, (Ruby 3.1, Rails 4) doesn't work because Ruby 3.1 isn't a combination supported by the Rails maintainers.  The only released combination that works is something not in your matrix (Ruby 3.1, Rails 7).  So Rails 7 would need to be added to the matrix (requiring exclusions for Ruby 2.5, 2.6) and Ruby 3.1 would need exclusions for Rails 4, 5, and 6.  So you could add Ruby 3.1 to the top level, but that would require another 5 exclusions on an already large exclusion list.

Regarding the second, please read the description above.  If you don't quote `'3.0` then GitHub Actions truncates it, and you get back the latest Ruby 3 - at time of writing Ruby 3.1.2.  That's not the desired outcome.